dazel/lib/src/bazelify/build.dart

```diff
@@ -56,7 +57,7 @@ class BuildFile {
     return _findHtmlEntryPoints(packageDir, searchDir)
         .map/*<DartWebApplication>*/((entryPoint) {
       return new DartWebApplication(
-        name: p.withoutExtension(entryPoint.htmlFile),
+        name: appPathToTarget(entryPoint.htmlFile),
```

Comment:
[nit] how about `appPathForTarget` and `targetForAppPath` etc. 'Path to' in the name makes it harder to understand that this is a bidirectional conversion.